Description: Upstream changes introduced in version 2.8.8-3ubuntu6~test2
Author: Andreas Moog <amoog@ubuntu.com>
Bug: http://sourceforge.net/support/tracker.php?aid=3024423
Bug-Ubuntu: https://launchpad.net/bugs/392503

--- xchat-2.8.8.orig/src/fe-gtk/menu.c
+++ xchat-2.8.8/src/fe-gtk/menu.c
@@ -2215,8 +2215,10 @@ togitem:
 													 mymenu[i].state);*/
 			if (mymenu[i].key != 0)
 				gtk_widget_add_accelerator (item, "activate", accel_group,
-									mymenu[i].key, mymenu[i].id == MENU_ID_AWAY ?
-									away_mask : GDK_CONTROL_MASK, GTK_ACCEL_VISIBLE);
+									mymenu[i].key,
+									mymenu[i].id == MENU_ID_AWAY ? away_mask :
+									mymenu[i].key == GDK_F9 ? 0 :
+									GDK_CONTROL_MASK, GTK_ACCEL_VISIBLE);
 			if (mymenu[i].callback)
 				g_signal_connect (G_OBJECT (item), "toggled",
 										G_CALLBACK (mymenu[i].callback), 0);
